.github/scripts/.helm-tests/default/result.yaml

```diff
@@ -11393,7 +11393,7 @@ spec:
         - name: CERT_MANAGER_ENABLED
           value: "true"
         image: ghcr.io/keptn/lifecycle-operator:v0.0.0
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
I would expect this to be Never, as the lifecycleOperator.imagePullPolicy helm value is set to never in the values.yaml file